va/http_test.go

```diff
@@ -838,51 +915,33 @@ func TestFetchHTTP(t *testing.T) {
 			},
 		},
 		{
-			Name: "Connecting to bad port",
```

Comment:
Removing this test. What it really tested was a doubled port: `example.com:80:80`. This is no longer structurally possible now that we more fully separate the port from the hostname inside the VA.